packages/changed-elements-react/src/widgets/FeedbackButton.scss

```diff
@@ -0,0 +1,21 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+.changed-elems-feedback-btn {
+  margin: 0 auto 50px;
+  overflow: hidden;
+  border-radius: 50px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  .changed-elems-feedback-content-wrapper {
+    display: flex;
+    alignitems: center;
```

Comment:
This is not a valid CSS property. Should be `align-items` but it seems the layout works without specifying alignment.